minio/nginx/nginx.conf

```diff
@@ -0,0 +1,73 @@
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
```

Comment:
See comment about journald logging.

Reply:
Don't we still need log paths configured inside the container?
Hopefully the nginx container will redirect them to standard output/error when the path is not mounted, or maybe my understanding is wrong...

Reply:
I would suggest that we do nothing specific for logs (appart from logging to the correct driver that is journald). This way we do not have to handle the rotation.

Reply:
Sure,  do you think the access / error . log entries in the configuration shall be removed?

Reply:
if they are not present in the default configuration file, then yes.

Reply:
Only the default configuration has access / error logs definitions now.
Let me know if there's anything you would like to add for this patch.